ReleaseTooling/Sources/ZipBuilder/main.swift

```diff
@@ -79,6 +79,12 @@ struct ZipBuilderTool: ParsableCommand {
         help: ArgumentHelp("A flag to indicate keeping (not deleting) the build artifacts."))
   var keepBuildArtifacts: Bool
 
+  /// Flag to skip building the Catalyst slices.
+  @Flag(default: false,
+        inversion: .prefixedNo,
+        help: ArgumentHelp("A flag to indicate keeping (not deleting) the build artifacts."))
+  var skipCatalyst: Bool
```

Comment:
Should this be something like `includeCatalyst` to avoid a double negative of `no-skip-catalyst`?

Reply:
Done.